pymc/sampling_jax.py

```diff
@@ -188,6 +188,45 @@ def sample_blackjax_nuts(
     chain_method="parallel",
     idata_kwargs=None,
 ):
+    """Draw samples from the posterior using the NUTS method from the blackjax library.
+
+    Parameters
```

Comment:
Can you update with the advise on https://pymc-data-umbrella.xyz/en/latest/sprint/docstring_tutorial.html#edit-the-docstring and I'll take a 2nd look later?

Reply:
I must be missing something but I think I'm style compliant. I actually copied the docstrings from `sample` as my starting point.

Reply:
Basically none of the docstrings are currently style compliant, and most aren't rendered correctly either because of that. We have an issue open for this: https://github.com/pymc-devs/pymc/issues/5459. I can give a quick go at one of the docstrings. I think you'll also need to rebase on main for the docs preview to work and CI to pass.

Reply:
added some comments, but they are not exaustive

Reply:
Oh ok it was how I'm specifying defaults and optional.